cdap-spark-core2_2.11/pom.xml

```diff
@@ -378,7 +378,7 @@
           <plugin>
             <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-shade-plugin</artifactId>
-            <version>2.4.3</version>
+            <version>3.2.1</version>
```

Comment:
Why not put this in the properties of the top level pom.xml?

Reply:
Traditionally we didn't have maven plugin version defined as properties. Ideally should be done through plugin management. Maybe worth a next round of refactoring. For this PR, the aim is to just get rid of mvn warnings.